bycycle/tests/test_plts_utils.py

```diff
@@ -0,0 +1,47 @@
+"""Tests plotting utilities."""
+
+import numpy as np
+import pandas as pd
+from bycycle.plts.utils import apply_tlims, get_extrema
+
+###################################################################################################
+###################################################################################################
+
+def test_apply_tlims(sim_args):
+    """Test apply tlims to signal and dataframe."""
+
+    df = sim_args['df']
+    sig = sim_args['sig']
+    fs = sim_args['fs']
+
+    times = np.arange(0, len(sig) / fs, 1 / fs)
+    tlims = (1, 2)
+
+    df_short, sig_short, times_short = apply_tlims(df, sig, times, fs, tlims)
+
+    print(df_short['sample_next_trough'].values)
```

Comment:
I presume these are a holdover from exploring the testing - but presumably we don't need the print lines?